features/built_in_matchers/all.feature

```diff
@@ -30,13 +30,18 @@ Feature: `all` matcher
         it { is_expected.to all( be_a(String) ) }
         it { is_expected.to all( be > 2 ) }
       end
+
+      RSpec.describe [1, 'a', 5] do
+        it { is_expected.to all( be < 10 ) }
+      end
```

Comment:
Sure, this is why is in a separate commit. I found this one, when developing `any`.